### DISCUSSION TITLE
win 10 start menu tiles

### DISCUSSION THREAD

**Discussion by rmachnowski2000** [valinet/ExplorerPatcher#3118] [Ideas]
a way to hide the tiles and just show the apps would be awesome....also nice product thanks

**Comment by pyrates999** [comment]
If you prefer the start menu from windows 7, install open shell.  It will have only the apps and no live tiles.  I prefer it.

If not, check in EP properties application under Start Menu.

Please also post windows version and build.  You can find this by running winver.
